File: Debug/Python/4.py
# To count the number of vowels

def count_vowels(string):
    vowels = {'a', 'e', 'i', 'o', 'u'} 
    vowel_counts = {}
    for char in string.lower():
        if char in vowels:
            vowel_counts[char] += vowel_counts
    return vowel_counts

input_string = "Hello World"
vowel_counts = count_vowels(input_string)
print("Vowel counts:", vowel_counts)

File: Debug/C++/4.cpp
// To count the number of vowels

#include <iostream>
#include <unordered_map>
#include <string>
#include <cctype>

std::unordered_map<char, int> countVowels(std::string str)
{
    std::unordered_map<char, int> vowelCounts;
    std::string vowels = "aeiou";
    for (char c : str)
    {
        if (vowels.find(std::toupper(c)) == std::string : npos)
        {
        }
    }
    return vowelCounts;
}

int main()
{
    std::string inputString = "Hello World";
    std::unordered_map<char, int> vowelCounts = countVowels(inputString);
    std::cout << "Vowel counts: ";
    for (auto pair : vowelCounts)
    {
        std::cout << pair.second << ":" << pair.first << " ";
    }
    std::cout << std::endl;
    return 0;
}
